tasks/watch.cr

```diff
@@ -41,18 +43,31 @@ module Sentry
 
       self.successful_compilations += 1
       if reload_browser?
-        reload_or_start_browser_sync
+        begin
+          reload_or_start_browser_sync
+        rescue Sentry::BrowserSyncPortInUseError
+          puts "Another process is currently using the BrowserSync port #{BROWSERSYNC_PORT}".colorize(:red)
```

Comment:
Any ideas on how we could make it easier for someone to figure out what process is using it or what to do? 

I'm also ok with this since it's better than not letting the user now, it'd just be super cool if there was a simple command they could run to find/stop processes running on a certain port. But I'm not aware of such a command

Reply:
@paulcsmith What are your thoughts on this output for the pretty error message? 
<img width="1481" alt="screen shot 2018-08-01 at 2 47 57 pm" src="https://user-images.githubusercontent.com/2391/43550981-3b84d2bc-959a-11e8-982d-10ef6b640121.png">